lib/tictactoe.rb

```diff
@@ -0,0 +1,10 @@
+class TicTacToe
+  def run
+    puts "Welcome to Tic-Tac-Toe"
+    true
+  end
+end
+
+if $PROGRAM_NAME == __FILE__
+  TicTacToe.new.run
+end
```

Comment:
Alternatively, we could put this into a script in the root of the repo or in a `bin` directory. This is ok too though.